src/Pact/Types/Lang.hs

```diff
@@ -57,7 +57,7 @@ module Pact.Types.Lang
    Meta(..),
```

Comment:
Likewise this module should remain unchanged. Definitely do not want Lang.hs depending on Pact.Analyze.Types. For `unTypeName` just use `asString`.